tiny_tds.gemspec

```diff
@@ -19,9 +19,10 @@ Gem::Specification.new do |s|
   s.extensions    = ['ext/tiny_tds/extconf.rb']
   s.license       = 'MIT'
   s.required_ruby_version = '>= 2.0.0'
-  s.add_runtime_dependency     'mini_portile2', '~> 2.0'
+  s.metadata['msys2_mingw_dependencies'] = 'freetds'
```

Comment:
This is described here: https://github.com/oneclick/rubyinstaller2/wiki/For-gem-developers#msys2-library-dependency